framework/include/utils/MooseTypes.h

```diff
@@ -678,6 +678,8 @@ enum EigenSolveType
   EST_JACOBI_DAVIDSON, ///< Jacobi-Davidson
   EST_NONLINEAR_POWER, ///< Nonlinear inverse power
   EST_NEWTON,          ///< Newton-based eigen solver
```

Comment:
Should change to `Newton Krylov with the full matrix assembled` now.

Reply:
Yes, we should make it consistent with regular nonlinear system.

Reply:
This comment has not been addressed.